hazelcast/src/test/java/com/hazelcast/security/SecurityWithoutEnterpriseTest.java

```diff
@@ -0,0 +1,25 @@
+package com.hazelcast.security;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.SecurityConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.io.IOException;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(QuickTest.class)
+public class SecurityWithoutEnterpriseTest extends HazelcastTestSupport {
+
+    @Test(expected = IllegalStateException.class)
+    public void test() throws IOException {
+        Config config = new Config();
+        config.setSecurityConfig(new SecurityConfig().setEnabled(true));
+        Hazelcast.newHazelcastInstance(config);
```

Comment:
Does this work with a mocked Hazelcast instance? This could make the test a `ParallelTest` then.